### PR TITLE
Configure BrowserID login/logout.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
PR updated. Open a new PR because Travis is grumpy. Build is passing now.

**Comment by johngian** [comment]
@akatsoulas r?

File: marketpulse/base/templates/base.html [hunk 1]
```diff
@@ -74,7 +82,11 @@ <h2 class="hidden-xs">
     <script src="{{ static('js/libs/bootstrap.min.js') }}"></script>
     <script src="{{ static('js/site.js') }}"></script>
     <script src="//mozorg.cdn.mozilla.net/tabzilla/tabzilla.js"></script>
+    <script src="https://login.persona.org/include.js"></script>
+    <script src="{{ static('browserid/api.js') }}"></script>
+    <script src="{{ static('browserid/browserid.js') }}"></script>
```
Comment (akatsoulas) [inline marketpulse/base/templates/base.html]: There is no need to include the two js libraries, you can use `{{ browserid_js }} instead after the js block.

**Comment by akatsoulas** [comment]
r+wc :bee:

File: marketpulse/settings/base.py [hunk 2]
```diff
@@ -154,8 +160,11 @@
 # Django-browserid settings
 AUTHENTICATION_BACKENDS = ('django.contrib.auth.backends.ModelBackend',
                            'marketpulse.auth.backend.MozilliansAuthBackend')
+
 BROWSERID_VERIFY_CLASS = 'marketpulse.auth.backend.BrowserIDVerify'
-BROWSERID_AUDIENCES = [SITE_URL]
+BROWSERID_AUDIENCES = config('BROWSERID_AUDIENCES', default=None)
```
Comment (akatsoulas) [inline marketpulse/settings/base.py]: This should be a list.

**Pull request merged**